### DISCUSSION TITLE
onDisconnect function take time to be executed

### DISCUSSION THREAD

**Discussion by ghost** [geckosio/geckos.io#199] [General]
Sorry, again a question from me lol.

Anyways, i'm trying to detect players disconnections, and immediatly delete the player entity when he leave the browser window, but 
the code that is in the onDisconnect function on server side is only executed like 10 seconds after closing the window.

Any idea ?

**Comment by ghost** [comment]
Seem to be an issue on node-datachannel

**Comment by aka-blackboots** [comment]
Hello there, @DarePhoenix. I'm glad you mentioned it. Do you have any ideas on how you solved this? I'm relatively new to the networking scene.
